src/components/Person/Person.jsx

```diff
@@ -1 +1,41 @@
-// export function Person({ person }) {}
+import React from 'react';
+
+export function Person({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) {
+  const partner = sex === 'm'
+    ? `My wife's name is ${partnerName}`
+    : `My husband's name is ${partnerName}`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {' '}
+        {name}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          I am
+          {' '}
+          {age}
```

Comment:
```suggestion
          {`I am ${age}`}
```

here is more convenient way